doc/source/whatsnew/v0.23.0.txt

```diff
@@ -406,6 +403,18 @@ Indexing
 - Bug in :func:`MultiIndex.set_labels` which would cause casting (and potentially clipping) of the new labels if the ``level`` argument is not 0 or a list like [0, 1, ... ]  (:issue:`19057`)
 - Bug in ``str.extractall`` when there were no matches empty :class:`Index` was returned instead of appropriate :class:`MultiIndex` (:issue:`19034`)
 
+MultiIndex
+^^^^^^^^^^
+
+- Bug in :func:`MultiIndex.get_level_values` which would return an invalid index on level of ints with missing values (:issue:`17924`)
```

Comment:
does this cover all of the listed issue at the top of the PR? (it may have extras of course)

Reply:
Yes, with the exception of #18519 which not strictly speaking a bug.